src/core/unit.c

```diff
@@ -4360,6 +4360,52 @@ int unit_patch_contexts(Unit *u) {
         }
 
         cc = unit_get_cgroup_context(u);
+        bool is_user_service = SERVICE(u) && strstr(u->id, "user@");
+        if (cc && (SLICE(u) || is_user_service)) {
+                FILE *f = fopen("/sys/fs/cgroup/dmem.region.capacity", "r");
```

Comment:
in our codebase use "re"

Reply:
FWIW this entire commit is supposed to go away in a non-draft version, it's definitely not code that should be merged -- I elaborated a bit in the last paragraph of my PR. TL;DR I'd like to do something like this (fixing the limit to "max" for certain units/types of units and to "0" for others) within unit configurations but I'm not sure how the design for that should look like.